ivy-custom-errorpage/context.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ========================================================================
+  Configures Valves and Realms of the embedded Tomcat Webserver
+ ========================================================================
+
+ Please keep the context.xml file on the designer and engine in sync
+ to have the same settings on designer and engine
+ as this file is not deployed from the designer to the engine
+
+ See apache tomcat documentation for more information about context configuration:
+ https://tomcat.apache.org/tomcat-9.0-doc/config/context.html
+
+-->
+<Context antiResourceLocking="false" privileged="true" >
+
+	<!-- ====================== Resources ==================== -->
+	
+	<!--
+	  Warning:
+	  This flag MUST NOT be set to true on the Windows platform (or any other OS which does not have a 
+	  case sensitive filesystem), as it will disable case sensitivity checks, allowing JSP source code disclosure, 
+	  among other security problems.
+	  
+	  If you use the debian package or run in a docker container and you have some web.xml configurations (e.g. custom error-pages), 
+	  you need to set allowLinking to true.
+	 -->
+	<Resources allowLinking="true" />
+
+
```

Comment:
for demo purpose, I would remove all other unecessary comments about 'Cookies', 'Tomcat Valves', etc.